### PR TITLE
add build instructions and application requirements to readme

### LINKED ISSUE #2: Building and running tests
I'm a newbie to Swift.  How do I build and run the tests?

### PR DESCRIPTION
**Pull request by NicoleCarpenter**
@mnewsome @zachmokahn fixes #2

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,36 @@
...
+Then run the application. 
+
+```swift
+.build/debug/Application
+``` 
+
+
+## Running the Tests
+
+Testing is accomplished using [Swiftest](https://github.com/zachmokahn/Swiftest.git) (version 0.1.8). To run the tests, make sure that the application has been compiled by running
```
Comment (zdavy) [inline README.md]: Swiftest isn't actually released as version 0.1.8, that's just my fork of it, with a release tagged 0.1.8. Also, you don't need to acknowledge the test runner. I'd remove this whole sentence:

> Testing is accomplished using [Swiftest](https://github.com/zachmokahn/Swiftest.git) (version 0.1.8).

**Comment by zdavy** [comment]
:+1: after you remove that one line

**Comment by NicoleCarpenter** [comment]
@zachmokahn fixed and didn't squash into initial commit.

**Comment by zdavy** [comment]
Cool you can go ahead and merge it after @mnewsome gives his approval

**Comment by mnewsome** [comment]
Nothing else jumps out at me at the moment.  :+1:

**Pull request merged**